backend/src/migrations/20230927022842_update_talent_hard_skills_table.ts

```diff
@@ -0,0 +1,33 @@
+import { type Knex } from 'knex';
+
+const TableName = {
+    TALENT_HARD_SKILLS: 'talent_hard_skills',
+} as const;
+
+const ColumnName = {
+    HARD_SKILL_ID: 'hard_skill_id',
+} as const;
+
+async function up(knex: Knex): Promise<void> {
+    return knex.schema.alterTable(TableName.TALENT_HARD_SKILLS, (table) => {
+        table.dropUnique([ColumnName.HARD_SKILL_ID]);
+    });
+}
+
+async function down(knex: Knex): Promise<void> {
+    await knex.raw(`
+  DELETE FROM ${TableName.TALENT_HARD_SKILLS}
+  WHERE ${ColumnName.HARD_SKILL_ID} IN (
+    SELECT ${ColumnName.HARD_SKILL_ID}
+    FROM ${TableName.TALENT_HARD_SKILLS}
+    GROUP BY ${ColumnName.HARD_SKILL_ID}
+    HAVING COUNT(*) > 1
+  )
+`);
+
+    return knex.schema.alterTable(TableName.TALENT_HARD_SKILLS, (table) => {
+        table.unique([ColumnName.HARD_SKILL_ID]);
+    });
+}
+
+export { down, up };
```

Comment:
Why we need hard skills migration in "Publish" task ?

Reply:
When I went through the talent onboarding process I got errors, so I couldn't finished the step. To "Publish task" I need to be field all my steps. That is why I updated talent_hard_skills table.
![image (1)](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/60103912/8bb3a871-db46-49aa-87f1-4df84782adc0)

Reply:
Ahh so that explains that the other day. I was wondering why I couldn't register hard skills.